scope.md

```diff
@@ -20,46 +20,58 @@ capabilities could help streamline the user experience:
 
 SPC is a tool for this last step.
 
-## Unique Features of SPC
+## Benefits and Unique Features of SPC
 
-* **Streamlined UX**. FIDO authentication provides a very good user
-experience. Our expectation is that the SPC "happy path" provides an
-even more streamlined experience. One important goal is to remove the
-need to redirect an enrolled user during a payment
+### Benefits
+
+* **Faster Authentication**. FIDO authentication provides a very good
+user experience. Our expectation is that the SPC "happy path" further
+accelerates strong customer authentication compared to other
+approaches such as one-time passwords. An important goal is to remove
+the need to redirect an enrolled user during a payment
 transaction. Another important goal is to speed up authentication
 compared to other approaches such as one-time passwords. A third goal
 is to create a consistent and trusted authentication experience across
 the Web.
 
 * **Scalable and Ubiquitous**. SPC supports streamlined authentication
-across merchant sites without additional enrollment. Note that with
-FIDO, the Relying Party that creates FIDO credentials is the only
-origin that can authenticate the user with those credentials. With
-SPC, any origin can authenticate the user using another Relying
-Party's credentials, provided that the authentication takes place
-within a known payment context. At the current time, Payment Request
-API establishes the payment context (but see [issue 56](https://github.com/w3c/secure-payment-confirmation/issues/56)). In addition, enabling payment
-service providers or others to authenticate the user can reduce the
-need to embed code provided by a Relying Party in a Web page, reducing
-security risks.
-
-* **Transaction Confirmation**. SPC generates cryptographic proof of
-the user's confirmation of transaction details. This is intended to
-help entities fulfill regulatory requirements (e.g., SCA for PSD2) or
-support other customer authentication use cases.
-
-* **Phishing-resistant**. Different entities may render the transaction
-details that the user confirms. In many cases the browser will display
-the transaction details in native UI, but SPC also supports display by
-secure hardware. Because the browser or secure hardware controls the
-display of transaction details, SPC is phishing-resistant.
-
-* **Simpler Front-end Deployment**. The browser (or secure hardware)
-manages the display of transaction confirmation, removing the need for
-other parties (e.g., issuing banks or payment apps) to do so.
+across merchant sites without additional enrollment.
+
+* **Designed to Meet Regulatory Requirements**. SPC helps entities
+fulfill regulatory requirements (e.g., strong customer authentication
+and transaction confirmation under PSD2) and other customer
+authentication use cases.
+
```

Comment:
I like this. It does seem to overlap with previous item somewhat. So suggest we remove the SCA references there.

Reply:
I've renamed the "faster" section to "Authentication Streamlined for Payment".
I've removed SCA from that one since covered under "regulatory"